tests/gx_test.py

```diff
@@ -330,6 +337,8 @@ def test_from_multilinestring(self) -> None:
 
         mt = MultiTrack(geometry=lines, ns="")
 
+        assert mt.__bool__() is True
```

Comment:
I think
```suggestion
        assert bool(mt)
```
is easier to read